### PR TITLE
Add PyLadies 2019 update blog post

### PR DESCRIPTION
**Pull request by lorenanicole**
# Overview

Add PyLadies 2019 update blog post

### Scope of work

- [x] Add PyLadies blog post with the AUg 15 update
- [x] Update about page to remove no longer supported communities
- [x] Test changes

### Questions

1. What else needs to be done beyond this? 

image
image
image

I tested everything locally and it looks great! Sooo ... we can merge in?

### CODE REVIEW

File: www/about/index.html [hunk 1]
```diff
@@ -12,11 +12,8 @@ <h3>Please join us!</h3>
 
             <p>Here's what to do:</p>
             <ol>
-                <li>Sign up for our newsletter to keep up with PyLadies news.</li>
                 <li>Join your local <a href="{{ get_url('/locations') }}">Meetup group</a> or start your own with our <a href="http://github.com/pyladies/pyladies-kit">PyLadies starter kit</a></li>
                 <li>Join our global <a href="https://plus.google.com/communities/108807002736066163985">Google+ community</a> to keep up with recent Pyladies conversations.</li>
```
Comment (Mariatta) [inline www/about/index.html]: Should this be removed too?

Comment (lorenanicole) [reply www/about/index.html]: Yikes yes let me do that!! Yay for PR review!

**Comment by lorenanicole** [comment]
Thanks for the review @Mariatta !

**Comment by willingc** [comment]
Awesome! Thanks for doing this.

**Pull request merged**